src/Common/REST/TEC/V1/Controller.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Controller for the TEC REST API.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\REST
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Common\REST\TEC\V1;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Common\REST\Controller as REST_Controller;
+
+/**
+ * Controller for the TEC REST API.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\REST
+ */
+class Controller extends Controller_Contract {
+	/**
+	 * The version of the REST API.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const VERSION = '1';
```

Comment:
If this is _always_ going to be an int, we should type it that way so no one puts in `1.5` or `1.0.1`

🤔 Probably best in these cases to put a long description for the var that tells devs what is allowable?

Reply:
PS: I'm really thinking of setting up expectations on the client side as well...